modes/MDS-passenger-services-state-machine-diagram.svg

```diff
@@ -0,0 +1,94 @@
+<svg width="1596" height="998" viewBox="0 0 1596 998" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Rename to just `passenger-services-state-machine-diagram.svg` for consistency with mode naming.

Reply:
Fixed in dc4ecaa